packages/NotificationCard/stories/NotificationCard.stories.js

```diff
@@ -0,0 +1,13 @@
+import { withKnobs } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import { getStoryName } from "storybook/storyTree";
+import Showcase from "./examples/Showcase";
+import Screener from "./examples/Screener";
+
+const storyName = getStoryName("NotificationCard");
+
+storiesOf(storyName, module)
```

Comment:
I won't insist on it for this PR, but I think we should be writing stories for new components in the new CSF.